lib/jekyll/page.rb

```diff
@@ -64,11 +62,11 @@ def initialize(site, base, dir, name)
     #
     # Returns the String destination directory.
     def dir
-      if url.end_with?(FORWARD_SLASH)
+      if url.end_with?("/")
         url
       else
         url_dir = File.dirname(url)
-        url_dir.end_with?(FORWARD_SLASH) ? url_dir : "#{url_dir}/"
+        url_dir.end_with?("/") ? url_dir : "#{url_dir}/"
```

Comment:
Should we replace `"#{url_dir}/"` with "#{url_dir}#{FORWARD_SLASH}"?

Reply:
er.. um.. I'm trying to do away with the constant here..
Why would I want to interpolate a single character..?